build/solr/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG MB_SOLR_VERSION=3.1.3
+FROM metabrainz/mb-solr:${MB_SOLR_VERSION}
+
+COPY [ "scripts/*", "/" ]
```

Comment:
Just a little nit pick: Wouldn't be ``/usr/local/bin/`` be a nicer place?

Reply:
Changed for scripts in both `search` and `musicbrainz` services with commit https://github.com/metabrainz/musicbrainz-docker/pull/137/commits/fa1a24e7c990ff151c2d13bcf56cbacbb1ce4e77.